src/screens/CompareModels/CompareModels.js

```diff
@@ -59,17 +86,85 @@ export function CompareModels({ match, location }) {
     });
   }
 
+  const [sortType, setSortType] = useState(SORT_TYPES.ALPHABETICAL);
+  const [filterTypeIntervention, setFilterTypeIntervention] = useState(
+    DEFAULT_INTERVENTION_FILTER,
+  );
+
+  const sortFunctionMap = {
+    [SORT_TYPES.ALPHABETICAL]: sortAlphabetical,
+    [SORT_TYPES.OVERWHELMED]: sortByDateOverwhelmed,
+  };
+
   function refresh() {
     setLeftUrl(leftText);
     setRightUrl(rightText);
     setQueryParams(leftText, rightText);
     setRefreshing(true);
   }
 
+  function sortAlphabetical(a, b) {
+    return a < b ? -1 : 1;
+  }
+
+  function sortByDateOverwhelmed(a, b) {
+    const overwhelmedDifferenceA = getDifferenceInDateOverwhelmed(a);
+    const overwhelmedDifferenceB = getDifferenceInDateOverwhelmed(b);
+    return overwhelmedDifferenceA > overwhelmedDifferenceB ? -1 : 1;
```

Comment:
If they're the same, should we be returning `0` instead of `1`?  Not sure, but as-is the order may be slightly non-deterministic.